celavi/pylca_celavi/pylca_opt_foreground.py

```diff
@@ -29,6 +29,8 @@ def preprocessing(year,state,df_static,dynamic_lci_filename,electricity_grid_spa
         cleaned process inventory merged with dynamic data
     pd.DataFrame    
         inventory with no product flows
+    columns=['flow name', 'flow unit', 'flow quantity',
```

Comment:
Make sure the indents match the previous lines, otherwise the HTML render will look wonky. Other than that, looks good.